atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/persistentlock/KvsBackedPersistentLockServiceTest.java

```diff
@@ -56,7 +56,8 @@ public void canAcquireLock() throws CheckAndSetException {
     @Test
     public void canReleaseLock() {
         LockEntry entry = lockStore.acquireBackupLock(TEST_REASON);
-        service.releaseLock(entry);
+        PersistentLockId entryId = ImmutablePersistentLockId.of(entry.instanceId());
```

Comment:
We should just hit acquire on the service. It's a bit weird interacting with the lock store directly